### ISSUE TITLE
[BUG] Qwen-14B-Chat-Int4 GPTQ model using vLLM. Got this warning: Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.

### ISSUE THREAD

**Issue by micronetboy** [QwenLM/Qwen#929]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

Qwen-14B-Chat-Int4 GPTQ model using vLLM.  Got the warning: Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.. How could we convert it to fast tokenizer?

use the following vllm_wrapper:
https://github.com/QwenLM/vllm-gptq/blob/main/tests/qwen/vllm_wrapper.py

The whole warning message:
WARNING 01-08 18:07:49 config.py:140] gptq quantization is not fully optimized yet. The speed can be slower than non-quantized models.
INFO 01-08 18:07:49 llm_engine.py:72] Initializing an LLM engine with config: model='../finetune/merged_14b_lora_gptq', tokenizer='../finetune/merged_14b_lora_gptq', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=True, dtype=torch.float16, max_seq_len=2048, download_dir=None, load_format=auto, tensor_parallel_size=1, quantization=gptq, seed=0)
WARNING 01-08 18:07:50 tokenizer.py:66] Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by jklj077** [comment]
Please ignore this; it is a false warning.

**Comment by houhailun** [comment]
我使用ray+vllm部署Qwen-72B-Chat-4bit，命令如下
ray start --head
ray start --address='xxx'

python -m vllm.entrypoints.openai.api_server  --trust-remote-code --model /home/bigdata/model/Qwen/Qwen-72B-Chat-Int4 --tensor-parallel-size 8 --seed 1234

输出以下信息后，任务卡死，等待十多分钟后仍没有反应
WARNING 03-14 17:48:30 tokenizer.py:64] Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.

**Comment by houhailun** [comment]
> Please ignore this; it is a false warning.
我使用ray+vllm部署Qwen-72B-Chat-4bit，命令如下
ray start --head
ray start --address='xxx'

python -m vllm.entrypoints.openai.api_server --trust-remote-code --model /home/bigdata/model/Qwen/Qwen-72B-Chat-Int4 --tensor-parallel-size 8 --seed 1234

输出以下信息后，任务卡死，等待十多分钟后仍没有反应
WARNING 03-14 17:48:30 tokenizer.py:64] Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.

**Comment by slliao445** [comment]
> > Please ignore this; it is a false warning.
> > 我使用ray+vllm部署Qwen-72B-Chat-4bit，命令如下
> > ray start --head
> > ray start --address='xxx'
> 
> python -m vllm.entrypoints.openai.api_server --trust-remote-code --model /home/bigdata/model/Qwen/Qwen-72B-Chat-Int4 --tensor-parallel-size 8 --seed 1234
> 
> 输出以下信息后，任务卡死，等待十多分钟后仍没有反应 WARNING 03-14 17:48:30 tokenizer.py:64] Using a slow tokenizer. This might cause a significant slowdown. Consider using a fast tokenizer instead.

我也是这样，请问有解决方法了吗
